fsharp-backend/src/LibExecutionStdLib/LibDate.fs

```diff
@@ -577,4 +591,21 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = SqlFunctionWithPrefixArgs("date_trunc", [ "'day'" ])
       previewable = Pure
+      deprecated = NotDeprecated }
+
+
+    // TODO: when we have timespans in Dark, deprecate this
+    // in favor of a fn which returns a timespan rather than seconds
+    { name = fn "Date" "subtract" 1
+      parameters = [ Param.make "end" TDate ""; Param.make "start" TDate "" ]
+      returnType = TInt
+      description = "Returns the difference of the two dates, in seconds"
+      fn =
+        (function
+        | _, [ DDate endDate; DDate startDate ] ->
+          let diff = (DDateTime.toInstant endDate) - (DDateTime.toInstant startDate)
+          diff.TotalSeconds |> System.Math.Round |> int64 |> DInt |> Ply
+        | _ -> incorrectArgs ())
+      sqlSpec = NotYetImplementedTODO
```

Comment:
maybe this is just '-'? I can test it out to see.

Reply:
the SQL here would be `EPOCH FROM (end - start)` but as far as I can see, we don't currently support such an expression (and it's beyond the scope of this PR to implement that in the sql compiler), so I'll leave it as-is, with a comment added.

Reply:
Hmm maybe `"EPOCH FROM", [ "-" ]` would work.

Reply:
Ah, it's `EXTRACT(EPOCH FROM (end - start))`, which would actually be more complicated.